common/persistence/serialization/serializer.go

```diff
@@ -121,21 +121,21 @@ type (
 		encodingType enumspb.EncodingType
 	}
 
-	serializerImpl struct {
+	SerializerImpl struct {
```

Comment:
`NewSerializer` is already exported. What's the use case for exporting the underlying implementation?